config/config.go

```diff
@@ -190,6 +195,7 @@ type Config struct {
 	AlertingConfig AlertingConfig  `yaml:"alerting,omitempty"`
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
+	StorageConfig  StorageConfig   `yaml:"storage_config,omitempty"`
```

Comment:
Let's just call this `storage`, to match the flag(s).

So the end yaml would look like:
```yaml
storage:
  exemplars:
    max_exemplars: ...
```